org.civicrm.reqangular/src/common/modules/directives.js

```diff
@@ -2,18 +2,21 @@ define([
   'common/angular',
   'common/decorators/ui-select/ui-select-choices',
   'common/decorators/angular-date/datepicker-addon',
+  'common/decorators/ui-bootstrap/uib-tabset',
+  'common/angularBootstrap',
   'common/ui-select',
   'common/modules/templates',
   'common/modules/controllers',
   'common/modules/services',
   'common/modules/apis'
-], function (angular, uiSelectChoicesCustomScrollBars, datepickerAddon) {
+], function (angular, uiSelectChoicesCustomScrollBars, datepickerAddon, uibTabset) {
```

Comment:
1. agreed
2. if `common/modules/api`/`common.apis` is needed by a controller, then we can move it to the `common.controllers` module